shared/cluster.go

```diff
@@ -233,7 +246,7 @@ func ParseNodes(print bool) ([]Node, error) {
 
 	res, err := RunCommandHost("kubectl get nodes --no-headers -o wide --kubeconfig=" + KubeConfigFile)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to run kubectl get nodes: %v", err)
```

Comment:
Dont we need to apply `ReturnLogError` here?

Reply:
i cant return the logger error here , bc we expect sometimes this returns nothing until has something to return.